test/unit/mongo_logger.test.ts

```diff
@@ -0,0 +1,204 @@
+import { expect } from 'chai';
+
+import { MongoLogger, SeverityLevel } from '../../src/mongo_logger';
+
+describe('class MongoLogger', function () {
+  describe('#constructor', function () {
+    it('treats the log destination value of stderr as case-insensitve', function () {
+      const loggerOptions = MongoLogger.resolveOptions({ MONGODB_LOG_PATH: 'STDERR' }, {});
```

Comment:
I think we are mix and matching the `resolveOptions` unit testing with the properties of the constructor itself; be precise - is it the options resolver that's responsible for the standardization of the option spelling or the constructor? if it's the options resolver, then this should be tested there, if it's the constructor, then you shouldn't pre-process what you are passing into it (i.e., provide the exact input to the constructor that you are testing)

Reply:
fixed and separated